Dockerfile

```diff
@@ -12,8 +12,12 @@ FROM base AS build
 
 WORKDIR /app
 COPY . .
+
+# delete package-lock.json - more info https://github.com/asyncapi/.github/issues/123
+# delete that line and install by `npm ci` when mentioned issue will be resolved
+RUN rm -rf package-lock.json
```

Comment:
Can we create a follow up issue so we can fix this once https://github.com/asyncapi/.github/issues/123 is solved (I hope it does soon)? Removing the package-lock.json file here has unexpected behavior as we don't control which version is gonna be installed of each package if they are not pinned to an specific version...

Reply:
I'm also against but sometimes switching from v1 to v2 (of packageLock version) can create errors, so bad and so not good 😆  but ok, we can go with that `package-lock.json`

Reply:
@smoya changed :)

Reply:
And also issue is created https://github.com/asyncapi/server-api/issues/36

Reply:
Oh no, it's not switching to one or another version something that scares me, no.
The fact we are not downloading exactly the same dependency versions as expected unless they are pinned to an exact version... this scares me 😆

Reply:
@smoya So current changes are ok for you or we should hardcode version of the dependencies?

Reply:
In fact, I thing this change should not be added into this PR but to a new one so:

- We don't hide side effects. This change is totally unexpected according to the name of the final commit that will be merged.
- We can quickly and easily revert the changes as they will be isolated.

Reply:
Reverted

Reply:
@smoya Created new PR https://github.com/asyncapi/server-api/pull/37